BoardConfig.mk

```diff
@@ -164,6 +164,8 @@ BOARD_ROOT_EXTRA_FOLDERS := firmware persist
 TARGET_USERIMAGES_USE_EXT4 := true
 TARGET_USERIMAGES_USE_F2FS := true
 TARGET_USES_MKE2FS := true
+TARGET_KERNEL_HAVE_EXFAT := true
+TARGET_EXFAT_DRIVER := exfat
```

Comment:
Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118

Reply:
> Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118

`TARGET_KERNEL_HAVE_EXFAT := true` is needed, please do a conflict resolution before merging, or let me know if it'd be better if I send this PR without the `TARGET_EXFAT_DRIVER := exfat` flag.

Reply:
> > Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118
> 
> `TARGET_KERNEL_HAVE_EXFAT := true` is needed, please do a conflict resolution before merging, or let me know if it'd be better if I send this PR without the `TARGET_EXFAT_DRIVER := exfat` flag.

Resend